templates/mariadb_centos.repo.j2

```diff
@@ -0,0 +1,7 @@
+# MariaDB CentOS {{ ansible_distribution_major_version|int }} repository list
```

Comment:
we need to make this it's own role.  See the other ansible roles in our repo that do just this.  Use: https://github.com/nexcess/ansible-role-repo-mariadb